static/app/views/alerts/wizard/panelContent.tsx

```diff
@@ -147,4 +147,64 @@ export const AlertWizardPanelContent: Record<AlertType, PanelContent> = {
     ],
     illustration: diagramCrashFreeUsers,
   },
+  app_start_cold: {
+    description: t('App cold start metric track when your app takes too long to launch.'),
+    examples: [
+      t(
+        'On iOS, Apple recommends your app take at most 400ms to render the first frame.'
+      ),
+      t(
+        'On Android, the Google Play console warns you when a cold start takes longer than five seconds or a warm start longer than two seconds.'
+      ),
+    ],
+    illustration: diagramCustom,
+  },
+  app_start_warm: {
+    description: t(
+      'App warm start metric track when your app takes does not too long to launch.'
+    ),
+    examples: [
+      t(
+        'On iOS, Apple recommends your app take at most 400ms to render the first frame.'
+      ),
+      t(
+        'On Android, the Google Play console warns you when a cold start takes longer than five seconds or a warm start longer than two seconds.'
+      ),
+    ],
+    illustration: diagramCustom,
+  },
+  frames_slow: {
+    description: t(
+      'Using 60 fps, slow frames are frames that take more time than necessary to render.'
```

Comment:
```suggestion
      'Slow frames are frames that take more time than necessary to render.'
```

Reply:
I would have made the same suggestion. Mentioning 60fps doesn't make sense. Why did you resolve this @dashed?